cmd/segment-reaper/observer.go

```diff
@@ -0,0 +1,160 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package main
+
+import (
+	"context"
+	"encoding/csv"
+	"strconv"
+
+	"github.com/gogo/protobuf/proto"
+	"github.com/zeebo/errs"
+	"go.uber.org/zap"
+
+	"storj.io/storj/pkg/pb"
+	"storj.io/storj/pkg/storj"
+	"storj.io/storj/satellite/metainfo"
+)
+
+// object represents object with segments.
+type object struct {
+	// TODO verify if we have more than 64 segments for object in network
+	segments uint64
```

Comment:
should we start using `bitmask` with this or next PR?

Reply:
Yes, we have, but I prefer a second PR, this one was created before the `bitmask` type existence and I prefer not to add more stuff o this one.